Classes/ObjectExplorers/Sections/Shortcuts/FLEXWindowShortcuts.m

```diff
@@ -0,0 +1,45 @@
+//
+//  FLEXWindowShortcuts.m
+//  FLEX
+//
+//  Created by AnthoPak on 26/09/2022.
+//
+
+#import "FLEXWindowShortcuts.h"
+#import "FLEXShortcut.h"
+#import "FLEXAlert.h"
+#import "FLEXObjectExplorerViewController.h"
+
+@implementation FLEXWindowShortcuts
+
+#pragma mark - Overrides
+
++ (instancetype)forObject:(UIView *)view {
+    return [self forObject:view additionalRows:@[
+        [FLEXActionShortcut title:@"Animations Speed" subtitle:^NSString *(UIWindow *window) {
+            return [NSString stringWithFormat:@"Current speed: %.2f", window.layer.speed];
+        } selectionHandler:^(UIViewController *host, UIWindow *window) {
+            [FLEXAlert makeAlert:^(FLEXAlert *make) {
+                make.title(@"Change Animations Speed");
+                make.message([NSString stringWithFormat:@"Current speed: %.2f", window.layer.speed]);
+                make.configuredTextField(^(UITextField * _Nonnull textField) {
+                    textField.placeholder = @"Speed value";
+                    textField.keyboardType = UIKeyboardTypeDecimalPad;
+                });
+                
+                make.button(@"OK").handler(^(NSArray<NSString *> *strings) {
+                    CGFloat speedValue = strings.firstObject.floatValue;
+                    window.layer.speed = speedValue;
+
+                    // Refresh the host view controller to update the shortcut subtitle, reflecting current speed
+                    [(FLEXObjectExplorerViewController *)host reloadData];
```

Comment:
Hmmm I don't like this but I guess that means I need a way to reload the item itself from this callback
```suggestion
                    // TODO: this shouldn't be necessary
                    [(FLEXObjectExplorerViewController *)host reloadData];
```